code/game/objects/items/trader.dm

```diff
@@ -148,3 +148,502 @@
+			if(findtext(speech.message, "standard") || findtext(speech.message, "sprinkle") || findtext(speech.message, "traditional"))
+				dispense_path = /obj/item/weapon/reagent_containers/food/snacks/donut/normal
+				sleep(3)
+				say("SPRINKLE.")
+			else if(findtext(speech.message, "jelly") || findtext(speech.message, "berry") || findtext(speech.message, "juicy"))
+				dispense_path = /obj/item/weapon/reagent_containers/food/snacks/donut/jelly
+				sleep(3)
+				say("JELLY.")
+			else if(findtext(speech.message, "chaos") || findtext(speech.message, "gump") || findtext(speech.message, "two-face"))
+				dispense_path = /obj/item/weapon/reagent_containers/food/snacks/donut/chaos
+				sleep(3)
+				say("CHAOS.")
+			else if(findtext(speech.message, "favorite") || findtext(speech.message, "4Kids") || findtext(speech.message, "rice"))
+				dispense_path = /obj/item/weapon/reagent_containers/food/snacks/riceball
+				sleep(3)
+				say("FAVORITE.")
+		if(dispense_path)
+			var/obj/item/I = new dispense_path(get_turf(src))
+			H.put_in_hands(I)
+			last_donut = world.timeofday
+
+//Security Skirt spritework is coutesy of TG, AGPL license
+/obj/item/clothing/under/securityskirt/elite
+	name = "elite security skirt"
+	desc = "For demonstrating who is in charge."
+	icon_state = "secskirt"
+	item_state = "r_suit"
+	_color = "secskirt"
+	armor = list(melee = 10, bullet = 10, laser = 10,energy = 0, bomb = 0, bio = 0, rad = 0)
+	clothing_flags = ONESIZEFITSALL
+	siemens_coefficient = 0.9
+	species_fit = list(GREY_SHAPED) //Unlike normal skirts this is not VOX_SHAPED
+	body_parts_covered = FULL_TORSO|ARMS
+
+/obj/item/clothing/under/securityskirt/elite/equipped(var/mob/user, var/slot)
+	..()
+	processing_objects += src
+
+/obj/item/clothing/under/securityskirt/elite/unequipped(mob/user, var/from_slot = null)
+	processing_objects -= src
+	..()
+
+/obj/item/clothing/under/securityskirt/elite/process()
+	if(prob(1))
+		if(ishuman(loc))
+			var/mob/living/carbon/human/H = loc
+			if(!(H.wear_suit && H.wear_suit.body_parts_covered & LEGS)) //It doesn't make sense to swish about if it's covered under something
+				H.visible_message("<span class='warning'>[H]'s [src] swishes threateningly.</span>",
+				"Your [src] fills you with confidence.",
+				"Something cracks like a whip.")
+				H.reagents.add_reagent(PARACETAMOL,1)
+
+/obj/item/weapon/ram_kit
+	name = "battering ram drop-leaf kit"
+	desc = "A device so ingenius there is no way the Vox invented it. Exploits volt-induced superposition to allow battering ram to fold into itself."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "modkit"
+	flags = FPRINT
+	siemens_coefficient = 0
+	w_class = W_CLASS_SMALL
+	origin_tech = Tc_COMBAT + "=5"
+
+/obj/structure/largecrate/secure
+	name = "security livestock crate"
+	desc = "An access-locked crate containing a security wolf. Handlers are responsible for obedience: wolves require regular meat or they will lash out at small animals and, if desperate, humans."
```

Comment:
You could just make a wolf reskin that looks like a dog and call it K9 unit

Reply:
I'll do it if you find someone to provide spritework

Reply:
just use sasha's icon